.github/actions/firebase-distribution/action.yml

```
@@ -0,0 +1,22 @@
+    - run: echo "Hi, I am a Calculator action!"
+      shell: bash
+    - run: echo "x = ${{ inputs.x }}, y = ${{ inputs.y }}"
+      shell: bash
+    - id: plus
+      run: echo "result=$((${{ inputs.x }} + ${{ inputs.y }}))" >> $GITHUB_OUTPUT
+      shell: bash
+    - id: minus
+      run: echo "result=$((${{ inputs.x }} - ${{ inputs.y }}))" >> $GITHUB_OUTPUT
+      shell: bash
+    - run: echo `plus:${{ steps.plus.outputs.result }}, minus:${{ steps.minus.outputs.result }}`
+      shell: bash
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)